src/app/contributors/contributors.service.ts

```diff
@@ -0,0 +1,15 @@
+import { HttpClient } from '@angular/common/http';
+import { inject, Injectable } from '@angular/core';
+import { Observable } from 'rxjs';
+import { ContributorsModel } from './contributors.model';
+
+@Injectable({ providedIn: 'root' })
+export class ContributorsService {
+  private http = inject(HttpClient);
+  private contributorsBasePath =
```

Comment:
nit: Base path implies this is used to build upon to make other paths. The actual base path here would be up to before `/contributors`, and if we ever needed to add something like a GitHub Stars counter later, we could reuse this.

```ts
private readonly basePath = 'https://api.github.com/repos/eneajaho/ngx-libs';
private readonly contributorsPath = this.basePath + '/contributors?q=contributions&order=asc';
```